pkgs/by-name/ri/rime-ls/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  librime,
+  rime-data,
+  gitUpdater,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "rime-ls";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "wlh320";
+    repo = "rime-ls";
+    rev = "v${version}";
+    hash = "sha256-ZqoRFIF3ehfEeTN+ZU+/PAzA4JyS1403+sqZdzwJHA8=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "librime-sys-0.1.0" = "sha256-zJShR0uaKH42RYjTfrBFLM19Jaz2r/4rNn9QIumwTfA=";
+    };
+  };
```

Comment:
Please check whether you can use the new fetcher (to avoid copying Cargo.lock here)

#349360

Reply:
It works here :)